### PR TITLE
Update outdated files in the dev-1.19-ko.3 branch

### LINKED ISSUE #24121: Outdated files in the dev-1.19-ko.3 branch
# This is a Bug Report
## Problem
Outdated files in the dev-1.19-ko.3 branch.

### 31 files to be modified 
1. [ ] content/en/docs/concepts/configuration/configmap.md 1 file changed, 1 insertion(+)
1. [ ] content/en/docs/concepts/configuration/manage-resources-containers.md 1 file changed, 1 deletion(-)
1. [ ] content/en/docs/concepts/configuration/secret.md 1 file changed, 20 insertions(+), 392 deletions(-)
1. [ ] content/en/docs/concepts/containers/images.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/overview/kubernetes-api.md 1 file changed, 50 insertions(+), 94 deletions(-)
1. [ ] content/en/docs/concepts/policy/resource-quotas.md 1 file changed, 13 insertions(+)
1. [ ] content/en/docs/concepts/scheduling-eviction/assign-pod-node.md 1 file changed, 1 insertion(+), 11 deletions(-)
1. [ ] content/en/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md 1 file changed, 2 insertions(+), 9 deletions(-)
1. [ ] content/en/docs/concepts/services-networking/endpoint-slices.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/services-networking/ingress.md 1 file changed, 63 insertions(+), 23 deletions(-)
1. [ ] content/en/docs/concepts/services-networking/service.md 1 file changed, 84 insertions(+), 73 deletions(-)
1. [ ] content/en/docs/concepts/storage/storage-classes.md 1 file changed, 15 insertions(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/storage/volumes.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md 1 file changed, 145 insertions(+), 56 deletions(-)
1. [ ] content/en/docs/contribute/suggesting-improvements.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/reference/kubectl/overview.md 1 file changed, 13 insertions(+), 6 deletions(-)
1. [ ] content/en/docs/reference/scheduling/policies.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/reference/using-api/api-overview.md 1 file changed, 62 insertions(+), 53 deletions(-)
1. [ ] content/en/docs/setup/best-practices/node-conformance.md 1 file changed, 6 deletions(-)
1. [ ] content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/administer-cluster/access-cluster-api.md 1 file changed, 4 insertions(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/administer-cluster/network-policy-provider/weave-network-policy.md 1 file changed, 2 insertions(+), 2 deletions(-)
1. [ ] content/en/docs/tasks/manage-kubernetes-objects/imperative-command.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/manage-kubernetes-objects/imperative-config.md 1 file changed, 12 insertions(+)
1. [ ] content/en/docs/tasks/manage-kubernetes-objects/kustomization.md 1 file changed, 1 insertion(+), 1 deletion(-)
1. [ ] content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md 1 file changed, 5 insertions(+), 5 deletions(-)
1. [ ] content/en/docs/tasks/run-application/horizontal-pod-autoscale.md 1 file changed, 7 insertions(+), 7 deletions(-)
1. [ ] content/en/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md 1 file changed, 3 insertions(+), 11 deletions(-)
1. [ ] content/en/partners/_index.html 1 file changed, 4 insertions(+), 49 deletions(-)

## Proposed Solution

Use `git diff` to check what is changed in the upstream. And apply the upstream changes manually 
to the `content/ko` of `dev-1.19-ko.3` branch.

For example:
```
# checkout `dev-1.19-ko.3`
...
# check what is updated in the upstream 
git diff dev-1.19-ko.2 dev-1.19-ko.3 -- content/en/docs/concepts/configuration/configmap.md
# apply changes to content/ko
vi content/ko/docs/concepts/configuration/configmap.md
...
# commit and push
...
# make PR to `dev-1.19-ko.3`
```

/language ko

### PR DESCRIPTION
**Pull request by pjhwa**
Closes #24121 

Update the following outdated files in the dev-1.19-ko.3 branch.

26 files are updated
- [x] content/en/docs/concepts/configuration/configmap.md 1 file changed, 1 insertion(+)
- [x] content/en/docs/concepts/configuration/manage-resources-containers.md 1 file changed, 1 deletion(-)
- [x] content/en/docs/concepts/configuration/secret.md 1 file changed, 20 insertions(+), 392 deletions(-)
- [x] content/en/docs/concepts/overview/kubernetes-api.md 1 file changed, 50 insertions(+), 94 deletions(-)
- [x] content/en/docs/concepts/policy/resource-quotas.md 1 file changed, 13 insertions(+)
- [x] content/en/docs/concepts/scheduling-eviction/assign-pod-node.md 1 file changed, 1 insertion(+), 11 deletions(-)
- [x] content/en/docs/concepts/scheduling-eviction/scheduler-perf-tuning.md 1 file changed, 2 insertions(+), 9 deletions(-)
- [x] content/en/docs/concepts/services-networking/endpoint-slices.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/services-networking/ingress.md 1 file changed, 63 insertions(+), 23 deletions(-)
- [x] content/en/docs/concepts/services-networking/service.md 1 file changed, 84 insertions(+), 73 deletions(-)
- [x] content/en/docs/concepts/storage/storage-classes.md 1 file changed, 15 insertions(+), 1 deletion(-)
- [x] content/en/docs/concepts/storage/volumes.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/concepts/workloads/pods/pod-topology-spread-constraints.md 1 file changed, 145 insertions(+), 56 deletions(-)
- [x] content/en/docs/contribute/suggesting-improvements.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/reference/kubectl/overview.md 1 file changed, 13 insertions(+), 6 deletions(-)
- [x] content/en/docs/reference/scheduling/policies.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/reference/using-api/api-overview.md 1 file changed, 62 insertions(+), 53 deletions(-)
- [x] content/en/docs/setup/best-practices/node-conformance.md 1 file changed, 6 deletions(-)
- [x] content/en/docs/tasks/access-application-cluster/web-ui-dashboard.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/administer-cluster/network-policy-provider/cilium-network-policy.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/administer-cluster/network-policy-provider/weave-network-policy.md 1 file changed, 2 insertions(+), 2 deletions(-)
- [x] content/en/docs/tasks/manage-kubernetes-objects/imperative-command.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/manage-kubernetes-objects/imperative-config.md 1 file changed, 12 insertions(+)
- [x] content/en/docs/tasks/manage-kubernetes-objects/kustomization.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tutorials/stateless-application/guestbook-logs-metrics-with-elk.md 1 file changed, 3 insertions(+), 11 deletions(-)
- [x] content/en/partners/_index.html 1 file changed, 4 insertions(+), 49 deletions(-)

5 files are already updated in Korean documents
- [x] content/en/docs/concepts/containers/images.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/concepts/workloads/pods/pod-lifecycle.md 1 file changed, 1 insertion(+), 1 deletion(-)
- [x] content/en/docs/tasks/administer-cluster/access-cluster-api.md 1 file changed, 4 insertions(+), 1 deletion(-)
- [x] content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md 1 file changed, 5 insertions(+), 5 deletions(-)
- [x] content/en/docs/tasks/run-application/horizontal-pod-autoscale.md 1 file changed, 7 insertions(+), 7 deletions(-)

### CODE REVIEW

File: content/ko/docs/reference/using-api/api-overview.md [hunk 1]
```diff
@@ -9,95 +9,105 @@ card:
...
-베타 기능을 사용해보고 피드백을 제공하자. 일단 베타가 끝나면, 실질적으로 더 많은 변경이 어렵다.
- {{< /note >}}
+  - 오브젝트에 대한 스키마나 문법이 다음 베타 또는 안정화 릴리스에서
+    호환되지 않는 방식으로 바뀔 수도 있다. 이런 경우, 다음 버전으로
+    이관할 수 있는 가이드가 제공된다. 스키마 변경은 API 오브젝트의 삭제, 편집 또는 재생성이
+    필요할 수도 있다. 편집 절차는 좀 생각해볼 필요가 있다.
+    이 기능에 의존하고 있는 애플리케이션은 다운타임이 필요할 수도 있다.
+  - 이 소프트웨어는 프로덕션 용도로 권장되지 않는다. 이후 여러 버전에서
+    호환되지 않는 변경 사항이 적용될 수 있다. 복수의 클러스터를 가지고 있어서
+    독립적으로 업그레이드할 수 있다면, 이런 제약에서 안심이 될 수도 있겠다.
```
Comment (seokho-son) [inline content/ko/docs/reference/using-api/api-overview.md:61]: 기존의 번역 사항이지만, 다소 어색해서 수정해보았습니다.

```suggestion
    독립적으로 업그레이드할 수 있다면, 이런 제약에서 벗어날 수도 있다.
```

can be upgraded independently, you may be able to relax this restriction.

File: content/ko/docs/reference/using-api/api-overview.md [hunk 2]
```diff
@@ -9,95 +9,105 @@ card:
...
+
+  {{< note >}}
+  베타 기능을 사용해보고 피드백을 제공하자. 일단 베타가 끝나면,
+  실질적으로 더 많은 변경이 어렵다.
```
Comment (seokho-son) [inline content/ko/docs/reference/using-api/api-overview.md:65]:
```suggestion
  베타 기능을 사용해보고 피드백을 제공하자. 기능이 베타 수준을 벗어난 이후에는
  실질적으로 더 많은 변경이 어렵다.
```
자연스럽게 수정해보았습니다.
  Try the beta features and provide feedback. After the features exit beta, it
  may not be practical to make more changes.

File: content/ko/docs/reference/using-api/api-overview.md [hunk 3]
```diff
@@ -9,95 +9,105 @@ card:
...
 
 ## API 그룹
 
-[*API 그룹*](https://git.k8s.io/community/contributors/design-proposals/api-machinery/api-group.md)은 쿠버네티스 API를 더 쉽게 확장하게 해준다. API 그룹은 REST 경로와 직렬화된 객체의 `apiVersion` 필드에 명시된다.
+[API 그룹](https://git.k8s.io/community/contributors/design-proposals/api-machinery/api-group.md)은
+쿠버네티스 API를 더 쉽게 확장하게 해준다.
+API 그룹은 REST 경로와 직렬화된 객체의 `apiVersion` 필드에
+명시된다.
 
 현재 다음과 같은 다양한 API 그룹이 사용되고 있다:
```
Comment (seokho-son) [inline content/ko/docs/reference/using-api/api-overview.md:79]:
```suggestion
현재 다음과 같은 다양한 API 그룹이 사용되고 있다.
```

File: content/ko/docs/reference/using-api/api-overview.md [hunk 4]
```diff
@@ -9,95 +9,105 @@ card:
...
-[사용자 정의 리소스](/ko/docs/concepts/extend-kubernetes/api-extension/custom-resources/)로 API를 확장하는 경우에는 다음 두 종류의 경로가 지원된다.
-
- - 기본적인 CRUD 요구에는
-   [커스텀리소스데피니션(CustomResourceDefinition)](/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/)
- - 쿠버네티스 API의 의미론적 전체 집합으로 사용자만의 Apiserver를 구현하려는 경우에는 [aggregator](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/api-machinery/aggregated-api-servers.md)
-
-
-## API 그룹 활성화 또는 비활성화 하기
+*  *핵심* (또는 *레거시* 라고 불리는) 그룹은 REST 경로 `/api/v1`에 있다.
+   핵심 그룹이 `apiVersion` 필드의 일부로 명시되지 않는다. 예를
```
Comment (seokho-son) [inline content/ko/docs/reference/using-api/api-overview.md:82]:
```suggestion
   핵심 그룹은 `apiVersion` 필드의 일부로 명시되지 않는다. 예를
```
The core group is not specified as part of the `apiVersion` field

**Review by seokho-son — COMMENTED** [review]
@pjhwa Thank you for the contribution for maintaining Korean contents !
I have some minor comments on this PR.

File: content/ko/docs/concepts/services-networking/endpoint-slices.md [hunk 5]
```diff
@@ -21,8 +21,8 @@ _엔드포인트슬라이스_ 는 쿠버네티스 클러스터 내의 네트워
 
 엔드포인트 API는 쿠버네티스에서 네트워크 엔드포인트를 추적하는
 간단하고 직접적인 방법을 제공한다. 불행하게도 쿠버네티스 클러스터와
-{{< glossary_tooltip text="서비스" term_id="service" >}}가 점점 더
-커짐에 따라, 이 API의 한계가 더욱 눈에 띄게
+{{< glossary_tooltip text="서비스" term_id="service" >}}가 더 많은 백엔드 파드로
+더 많은 트래픽을 처리하고 전송하도록 커짐에 따라, 이 API의 한계가 더욱 눈에 띄게
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/concepts/services-networking/endpoint-slices.md:25]: 반영하겠습니다.

File: content/ko/docs/reference/kubectl/overview.md [hunk 6]
```diff
@@ -125,29 +125,30 @@ kubectl [command] [TYPE] [NAME] [flags]
 
 다음 표에는 지원되는 모든 리소스 타입과 해당 약어가 나열되어 있다.
 
-(이 출력은 `kubectl api-resources` 에서 확인할 수 있으며, 쿠버네티스 1.13.3 부터 일치했다.)
+(이 출력은 `kubectl api-resources` 에서 확인할 수 있으며, 쿠버네티스 1.19.1 부터 일치했다.)
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/reference/kubectl/overview.md:128]: 반영하겠습니다.

File: content/ko/docs/reference/using-api/api-overview.md (hunk 1 above)
Comment (pjhwa) [reply content/ko/docs/reference/using-api/api-overview.md:61]: 반영하겠습니다.

File: content/ko/docs/reference/using-api/api-overview.md (hunk 2 above)
Comment (pjhwa) [reply content/ko/docs/reference/using-api/api-overview.md:65]: 반영하겠습니다.

File: content/ko/docs/reference/using-api/api-overview.md (hunk 3 above)
Comment (pjhwa) [reply content/ko/docs/reference/using-api/api-overview.md:79]: 반영하겠습니다.

File: content/ko/docs/reference/using-api/api-overview.md (hunk 4 above)
Comment (pjhwa) [reply content/ko/docs/reference/using-api/api-overview.md:82]: 반영하겠습니다.

File: content/ko/docs/concepts/services-networking/ingress.md [hunk 7]
```diff
@@ -27,15 +27,26 @@ weight: 40
...
+  service-->pod1[Pod];
+  service-->pod2[Pod];
+  end
+  classDef plain fill:#ddd,stroke:#fff,stroke-width:4px,color:#000;
+  classDef k8s fill:#326ce5,stroke:#fff,stroke-width:4px,color:#fff;
+  classDef cluster fill:#fff,stroke:#bbb,stroke-width:2px,color:#326ce5;
+  class ingress,service,pod1,pod2 k8s;
+  class client plain;
+  class cluster cluster;
+{{</ mermaid >}}
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/concepts/services-networking/ingress.md:46]: 일단 한글화를 해보겠습니다. 
출력되는 결과를 보시고 또 의견 부탁드립니다.

[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/concepts/services-networking/ingress.md]: https://deploy-preview-24151--k8s-dev-ko.netlify.app/ko/docs/concepts/services-networking/ingress/#%EC%9D%B8%EA%B7%B8%EB%A0%88%EC%8A%A4%EB%9E%80

File: content/ko/docs/concepts/services-networking/ingress.md [hunk 8]
```diff
@@ -272,10 +283,25 @@ test-ingress   external-lb   *       203.0.113.123   80      59s
...
+  service2-->pod3[Pod];
+  service2-->pod4[Pod];
+  end
+  classDef plain fill:#ddd,stroke:#fff,stroke-width:4px,color:#000;
+  classDef k8s fill:#326ce5,stroke:#fff,stroke-width:4px,color:#fff;
+  classDef cluster fill:#fff,stroke:#bbb,stroke-width:2px,color:#326ce5;
+  class ingress,service1,service2,pod1,pod2,pod3,pod4 k8s;
+  class client plain;
+  class cluster cluster;
+{{</ mermaid >}}
```
[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/concepts/services-networking/ingress.md]: https://deploy-preview-24151--k8s-dev-ko.netlify.app/ko/docs/concepts/services-networking/ingress/#%EA%B0%84%EB%8B%A8%ED%95%9C-%ED%8C%AC%EC%95%84%EC%9B%83-fanout

[the remark this answers is not in the document]
Comment (pjhwa) [reply content/ko/docs/concepts/services-networking/ingress.md]: 아래 '이름 기반의 가상 호스팅'에도 그림이 있어 함께 한글화하였습니다.
https://deploy-preview-24151--k8s-dev-ko.netlify.app/ko/docs/concepts/services-networking/ingress/#%EC%9D%B4%EB%A6%84-%EA%B8%B0%EB%B0%98%EC%9D%98-%EA%B0%80%EC%83%81-%ED%98%B8%EC%8A%A4%ED%8C%85

File: content/ko/docs/concepts/services-networking/ingress.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (seokho-son) [reply content/ko/docs/concepts/services-networking/ingress.md]: @pjhwa 감사합니다. 일단 문서 완성도가 높아지는 느낌이네요!

File: content/ko/docs/concepts/workloads/pods/pod-topology-spread-constraints.md [hunk 9]
```diff
@@ -30,13 +30,23 @@ node4   Ready    <none>   2m43s   v1.16.0   node=node4,zone=zoneB
...
+        n1(Node1)
+        n2(Node2)
+    end
+
+    classDef plain fill:#ddd,stroke:#fff,stroke-width:4px,color:#000;
+    classDef k8s fill:#326ce5,stroke:#fff,stroke-width:4px,color:#fff;
+    classDef cluster fill:#fff,stroke:#bbb,stroke-width:2px,color:#326ce5;
+    class n1,n2,n3,n4 k8s;
+    class zoneA,zoneB cluster;
+{{< /mermaid >}}
```
Comment (seokho-son) [inline content/ko/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:49]: @pjhwa 업데이트 감사합니다! 
content/ko/docs/concepts/workloads/pods/pod-topology-spread-constraints.md 에도

<mermaid>방식의 이미지들이 있는데, 한글화하는 경우,
예시 코드 상의 용어, 그림의 용어, 내용 상의 용어 등 일관성을 맞추는 작업이 필요해 보이네요.

해당 PR에서 수정해도 되겠지만, 별도의 PR을 여는 것도 방법일 것 같습니다. 참고하시기 바랍니다.. ^^

Comment (pjhwa) [reply content/ko/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:49]: 네, 그림의 용어랑 내용상의 용어를 맞추는 것이 필요할 것 같습니다. 
이 문서에서는 한글화를 한다고 하면, 'Pod' 정도가 아닐런지요? zoneA, Node1 등은 예시 코드 상에 표현되어 있는 부분이라...

Comment (seokho-son) [reply content/ko/docs/concepts/workloads/pods/pod-topology-spread-constraints.md:49]: @pjhwa 넵 ^^ 일단 outdated contents update 는 현재 상태로 마무리하시지요 ~~

**Comment by seokho-son** [comment]
감사합니다!
/lgtm
/approve

**Pull request merged**